testsuite/do_tests.sh

```diff
@@ -120,14 +120,16 @@ if test -d "${REPORTDIR}"; then
 fi
 mkdir "${REPORTDIR}"
 
-TEST_BASEDIR="${PREFIX}/share/doc/nest"
+TEST_BASEDIR="${PREFIX}/share/nest/testsuite"
```

Comment:
Single-quote escape slashes are extraneous on lines 44 and 48 (but I cannot place an in-line comment there!)